### PR TITLE
Added sql describe statement

### PR DESCRIPTION
**Pull request by ufierro**
I wanted to add the describe statement, it looks like it's meant to be handled as a simple SQL Result and this works just fine, but the response is just:
 ```[('id', 'INTEGER', True, 'PRIMARY KEY', False, True), ('name', 'VARCHAR[100]', True, 'NO', False, False)]```. 

Which is manageable as long as you know what each value is referring to but can be confusing otherwise

So that begs the question: Should this be a dict instead?
i.e. 

```
{ 'column': 'id', 'type': 'INTEGER', 'nullable': True, 'index': 'PRIMARY KEY', 'auto_increment': False, 'unique': True }
``` 

Or should a hint or additional value be returned?

### CODE REVIEW

**Comment by SimoneLazzaris** [comment]
In my opinion, that should be a map OR it can be an array of dataclass row descriptor: in this way it will not position-coded which is not intuitive and error prone

**Comment by jeroiraz** [comment]
> I wanted to add the describe statement, it looks like it's meant to be handled as a simple SQL Result and this works just fine, but the response is just: `[('id', 'INTEGER', True, 'PRIMARY KEY', False, True), ('name', 'VARCHAR[100]', True, 'NO', False, False)]`.
> 
> Which is manageable as long as you know what each value is referring to but can be confusing otherwise
> 
> So that begs the question: Should this be a map instead? i.e.
> 
> ```
> { 'column': 'id', 'type': 'INTEGER', 'nullable': True, 'index': 'PRIMARY KEY', 'auto_increment': False, 'unique': True }
> ```
> 
> Or should a hint or additional value be returned?

The columns in the result set describe the meaning of each positional value, as a normal query result.
e.g. immuclient is able to just render as a table:

```sql
immuclient>describe mytable
+----------+---------------+----------+---------------+----------------+--------+
|  COLUMN  |     TYPE      | NULLABLE |     INDEX     | AUTO INCREMENT | UNIQUE |
+----------+---------------+----------+---------------+----------------+--------+
| "id"     | "INTEGER"     | true     | "PRIMARY KEY" | true           | true   |
| "title"  | "VARCHAR[10]" | true     | "YES"         | false          | false  |
| "active" | "BOOLEAN"     | true     | "YES"         | false          | false  |
+----------+---------------+----------+---------------+----------------+--------+
```

**Comment by ufierro** [comment]
@jeroiraz The reason I bring this up is that within the Python SDK we go from

A response like this:
```rows {
  columns: "(defaultdb.testtable1816.id)"
  columns: "(defaultdb.testtable1816.name)"
  values {
    n: 1
  }
  values {
    s: "Joe"
  }
}
columns {
  name: "(defaultdb.testtable1816.id)"
  type: "INTEGER"
}
columns {
  name: "(defaultdb.testtable1816.name)"
  type: "VARCHAR"
}
```
To a return value like this: 
```
[(1, "Joe")]
```
Which seems to work just fine and provides a Pythonic way of handling query results; but omits the column names which is not something a describe statement should do.

My aim is to either keep both; describe and query as close as possible to each other but continue offering the same ease of use and maybe improve querying a bit without breaking anything

**Comment by ufierro** [comment]
@SimoneLazzaris  I made some more changes, I can do the same for queries if you think it's OK but it will not be backwards compatible

**Comment by SimoneLazzaris** [comment]
I think that the describe situation is different from a simple query: in a query, you can explicitly tell the sql engine which are the column you want, and in which order, so it's reasonable to get an array of (anonymous) tuples. In the `describe` scenario, I want to inspect a table, and python being python, I (user) expect some degree of introspection.

So I'd like to have an explicit dataclass like this:
```python
@dataclass
class ColumnDescription:
    name: string
    type: string
    nullable: bool
    index: bool
    autoincrement: bool
    unique: bool
```
and then have the result as an array of `ColumnDescription` instances. Or maybe provide that as an "high level" method.

**Comment by ufierro** [comment]
I like the idea of ColumnDescription, the only difference is the ```index``` field; I've set it to ```str``` to match the response

File: immudb/handler/sqldescribe.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+from dataclasses import dataclass
+from immudb.typeconv import sqlvalue_to_py
+
+from immudb.grpc import schema_pb2
+from immudb.grpc import schema_pb2_grpc
+from immudb.rootService import RootService
+
+
+@dataclass
```
Comment (SimoneLazzaris) [inline immudb/handler/sqldescribe.py:21]: Maybe it's better to move the dataclass definition with the others, in the `/datatypes.py` file?

**Review by SimoneLazzaris — COMMENTED** [review]
Seems fine, just maybe we can move the dataclass definition in the datatypes.py file?

File: immudb/handler/sqldescribe.py (hunk 1 above)
Comment (ufierro) [reply immudb/handler/sqldescribe.py]: Agreed

**Comment by ufierro** [comment]
moved dataclass over to datatypes.py and added a missing import to verifiedSet.py

**Pull request merged**